osc-rest-server/src/main/java/org/osc/core/broker/rest/server/api/VirtualizationConnectorApis.java

```diff
@@ -512,6 +511,40 @@ public Response updateVirtualSecurityPolicyBindings(@Context HttpHeaders headers
         return this.apiUtil.getResponseForBaseRequest(this.bindSecurityGroupService, bindRequest);
     }
 
+    @ApiOperation(value = "Set Security Group Bindings with Service Function Chain  (Openstack Only)",
+            notes = "Adds/Update/Remove Security Group Bindings to Security Function Services.<br/>"
+            		+ "To unbind a service function chain with security group pass null or empty value.<br/>"
+                    + "To Remove all policies, pass in empty json.<br/>"
+                    + "To update services binded to, pass in the updated list of services.<br/>",
+            response = BaseJobResponse.class)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Successful operation"),
+            @ApiResponse(code = 400, message = "In case of any error", response = ErrorCodeDto.class) })
+    @Path("/{vcId}/securityGroups/{sgId}/sfc/{sfcId}/bindings")
+    @PUT
+    public Response updateVirtualSecurityPolicyBindingsWithSfc(@Context HttpHeaders headers,
```

Comment:
you can create delete api /{vcId}/securityGroups/{sgId}/sfc/unbind

Reply:
Added new API for unbind sfc from Security group